ArdentHQ/dashbrd
# feat: show previous month winners

## Summary
Depends on https://github.com/ArdentHQ/dashbrd/pull/534
https://app.clickup.com/t/86dqnpma8

The implementation is based on the `has_won_at` field of collections. It retrieves all collections that have `has_won_at` set and sorts them (descending order) by date & vote count.

Additionally, a new seeder is added for winner collections to fill `has_won_at` fields on random collections.

## Testing

1. Run `composer db:live`, so that the new seeder for winner collections will run.
2. Open `/collections/collection-of-the-month` and you should be able to see the previous winners' tables of the last two years. 








## Checklist
